etna/transforms/math/statistics.py

```diff
@@ -136,6 +141,12 @@ class MeanTransform(WindowStatisticsTransform):
 
     .. math::
        MeanTransform(x_t) = \\sum_{i=1}^{window}{x_{t - i}\\cdot\\alpha^{i - 1}}
+
+    This transform, applied to non-regressor column, generates non-regressor column.
```

Comment:
I think we should wrap this in a warning block like this:
```
Warning
-------
```

For reference, look at `ChangePointsTrendTransform`.